flathub.json

```
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
+  "automerge-flathubbot-prs": true
```

```suggestion
  "only-arches": ["x86_64"]
```

we are phasing out the use of this option.